packages/react-reconciler/src/ReactFiberPerformanceTrack.js

```diff
@@ -182,14 +235,14 @@ export function logSuspenseBoundaryClientRendered(
         properties.push(['Error', message]);
       }
     }
-    performance.measure('Suspense', {
+    performance.measure(name, {
       start: startTime,
       end: endTime,
       detail: {
         devtools: {
           color: 'error',
           track: COMPONENTS_TRACK,
-          tooltipText: 'Hydration failed',
+          tooltipText: 'A life cycle or effect errored',
```

Comment:
I think both "lifecycle" and "life cycle" are correct, but our other errors don't use a space so you could be consistent here and save a char.

```suggestion
          tooltipText: 'A lifecycle or effect errored',
```

You could also consider being hooks-first and lead with "effect"

```suggestion
          tooltipText: 'An effect or lifecycle errored',
```